release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -3009,7 +3009,42 @@ link:https://access.redhat.com/solutions/6114681[{product-title} 4.6.34 containe
 
 With this update, users can now collect the `virt_platform` metric. The `virt_platform` metric is needed for the Insights Operator's rules to determine the virtual platform of a cluster. This information is stored in the Insights Operator archive of the `config/metric` file.  For more information, see link:https://bugzilla.redhat.com/show_bug.cgi?id=1965219[*BZ#1965219*].
 
-[id="ocp-44-6-34-upgrading"]
+[id="ocp-4-6-34-upgrading"]
 ==== Upgrading
 
-To upgrade an existing {product-title} 4.6 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
\ No newline at end of file
+To upgrade an existing {product-title} 4.6 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc#update-service-overview_updating-cluster-cli[Updating a cluster by using the CLI] for instructions.
+
+[id="ocp-4-6-35"]
+=== RHBA-2021:2410 - {product-title} 4.6.35 bug fix update
+
+Issued: 2021-06-22
+
+{product-title} release 4.6.35 is now available. The list of bug fixes that are included in the update is documented in the link:https://access.redhat.com/errata/RHBA-2021:2410[RHBA-2021:2410] advisory. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHBA-2021:2407[RHBA-2021:2407] advisory.
+
+Space precluded documenting all of the container images for this release in the advisory. See the following article for notes on the container images in this release:
+
+link:https://access.redhat.com/solutions/6128971[{product-title} 4.6.35 container image list]
+
+[id="ocp-4-6-35-features"]
+==== Features
+
+[id="ocp-4-6-35-cluster-operator-gatherer-enhancement"]
+===== Insights Operator enhancement
+
+With this update, ClusterOperator `Gather` can now collect logs from pods that are related to unhealthy operators, and/or pods that are living in the same namespace as the operator. This allows the Insights Operator to determine the namespace associated with the operator and include pod logs in the bundle. As a result, the Insights Operator is now collecting logs from unhealthy pods not only related to the operator, but from every pod that lives in the operator namespace.  
```

Comment:
Might just go "unhealthy Operators and pods"

Reply:
Maybe "living" -> "colocated"

Reply:
Although thoughout: s/operator/Operator/

Reply:
The way it's formatted, I can't tell if `Gather` is supposed to be the name of a cluster Operator or a command of something's. Poking around https://github.com/openshift/insights-operator it looks like might be considered a "method" or a "function"; if that's true, then maybe: "With this enhancement, the Insights Operator `Gather` method..."